### ISSUE TITLE
Are We Having Fun Yet? Stress and Exercise

### ISSUE THREAD

**Issue by nadjibandnaaima** [nadjibandnaaima/diets-for-quick-weight-loss#709]
<div dir="ltr" style="text-align: left;">
<span style="font-size: x-large;">Are We Having Fun Yet? Stress and Exercise</span><br>
<ins class="adsbygoogle" style="display: inline-block; width: 336px; height: 280px;" data-ad-client="ca-pub-2862230082201059" data-ad-slot="5129813120"></ins> 
//<![CDATA[
(adsbygoogle = window.adsbygoogle || []).push({});
//]]>
<br>
Like taxes, stress is a part of everyone&rsquo;s life.&nbsp; The &ldquo;experts&rdquo; tell us that some stress is good because without it there would be no motivation to do anything!&nbsp; However, if you are like me, serious stress and anxiety makes me feel bad physically &ndash; I am unable to eat. This can&rsquo;t be good!&nbsp;<br><br>
For most of us, stress is low-level most of the time.&nbsp; Our daily tasks and relationships give us a bit of stress but not major anxiety.&nbsp; The body&rsquo;s response to low-level stressors is designed to motivate us to action with a slightly raised heart rate and increased mental clarity &ndash;similar to the response to caffeine.&nbsp;&nbsp;<br><br>
In times where stress levels go higher, the responses are what many people call the &ldquo;fight or flight&rdquo; response.&nbsp; The body sends blood to the extremities to get ready for action!&nbsp;&nbsp; These responses are normal and good for you &ndash; except when the stress levels remain high for a long time.&nbsp; When the body perpetually stays in a stressed state, the systems begin to break down.&nbsp; Think of it like a car engine where the throttle is stuck and the engine is constantly &ldquo;revved up&rdquo;.&nbsp; Not only will it consume A LOT of gasoline, but the mechanical parts will wear out faster.<br><br>
So what to do to protect ourselves from this?&nbsp; RELAX of course&hellip;.and EXERCISE.&nbsp; Exercise helps us relax both physiologically and, if it&rsquo;s enjoyable, provides a mental escape from the stressor.&nbsp; The physiological effects of exercise include release of morphine-like chemicals (which include beta-endorphins) that give us a sense of well being.&nbsp; Have you ever heard the term Runner&rsquo;s High?&nbsp; This is caused by the release of &ldquo;endorphins&rdquo; into the blood.&nbsp; Since these chemicals stay in the blood for several hours after stopping the activity, the &ldquo;feel good&rdquo; feeling remains for a while.&nbsp;<br><br>
OK, so we know why exercise can help with anxiety and stress.&nbsp; But what are some of the ways regular exercise keeps us healthy in normal, low-level stress, times?<br><br>
Here&rsquo;s some from the top of my head:<br><br>
Using a daily dose of natural &ldquo;feel good&rdquo; chemicals is better than drugs or alcohol!!&nbsp;<br>
&nbsp;Looking fit, healthy and strong&nbsp; (It&rsquo;s ok to admit that looking good makes you feel good!)&nbsp;<br>
&nbsp;Feeling good from accomplishment and improvement.&nbsp;<br>
&nbsp;Escaping from the daily grind for an hour &ndash; change of scene.<br>
&nbsp;Keeping the body systems prepared to handle high-level stress.&nbsp;<br><br>
The key to lowering stress with exercise is that you ENJOY the exercise that you choose to do.&nbsp; If you hate going to the gym, it&rsquo;s not going to lower your stress &ndash; in fact it probably will add some!&nbsp; Also, studies are showing over and over that even low intensity exercise helps anxiety, so any activity you enjoy is good.&nbsp; For me, when I am under stress, nothing works better than a good sweaty dance session.&nbsp; The music, movement and high heart rate just melt away my troubles.&nbsp; Some of my clients even tell me that the sense of achievement from competitive sports makes them feel less stressed!<br><br>
The bottom line?&nbsp; To decrease the effects stress on your body, pick a fitness or sports activity you enjoy, do it regularly and HAVE FUN.&nbsp;<br><br>
</div>
<br><br>
via diets for quick weight loss http://ift.tt/1ml1DIp<br>
December 31, 2015 at 02:51PM
